discord-bot/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scripts": {
+    "build": "tsc --project tsconfig.json",
+    "start": "yarn run build && nodemon --watch src --exec ts-node src/index.ts"
+  },
+  "type": "module",
+  "dependencies": {
+    "discord.js": "^14.3.0",
+    "manifold-sdk": "2.0.1"
```

Comment:
I 100% would not depend on this unless you think that we should plan on owning this library and taking responsibility for it in the future.

Reply:
🪓